src/test/java/org/schemaspy/testcontainer/MysqlHTMLIT.java

```diff
@@ -0,0 +1,118 @@
+                    .assumeDockerIsPresent().withAssumptions(assumeDriverIsPresent())
+                    .withQueryString("?useSSL=false")
+                    .withInitScript("integrationTesting/dbScripts/mysql_html_implied_relationship.sql");
+
+    @BeforeClass
+    public static void generateHTML() throws Exception {
+        MySQLContainer container = jdbcContainerRule.getContainer();
+        String[] args = new String[]{
+                "-t", "mysql",
+                "-db", "test",
+                "-s", "test",
+                "-host", container.getContainerIpAddress() + ":" + String.valueOf(container.getMappedPort(3306)),
+                "-port", String.valueOf(container.getMappedPort(3306)),
+                "-u", container.getUsername(),
+                "-p", container.getPassword(),
+                "-o", "target/mysqlhtml",
+                "-connprops", "useSSL\\=false"
+        };
+        Main.main(args);
+    }
+
+    @Test
+    public void verifyXML() {
+        Diff d = DiffBuilder.compare(Input.fromURL(expectedXML))
+                .withTest(Input.fromFile("target/mysqlhtml/test.test.xml"))
+                .build();
+        assertThat(d.getDifferences()).isEmpty();
+    }
+
+    @Test
+    public void verifyDeletionOrder() throws IOException {
+        assertThat(Files.newInputStream(Paths.get("target/mysqlhtml/deletionOrder.txt"), StandardOpenOption.READ)).hasSameContentAs(expectedDeletionOrder.openStream());
+    }
+
+    @Test
+    public void verifyInsertionOrder() throws IOException {
+        assertThat(Files.newInputStream(Paths.get("target/mysqlhtml/insertionOrder.txt"), StandardOpenOption.READ)).hasSameContentAs(expectedInsertionOrder.openStream());
+    }
+
+    @Test
+    public void producesSameContet() throws IOException {
+        String target = "target/mysqlhtml";
+        Path expectedPath = Paths.get("src/test/resources/integrationTesting/expecting/mysqlhtml");
+        List<Path> expectations;
+        try (Stream<Path> pathStream = Files.find(expectedPath, 5, (p, a) -> a.isRegularFile())) {
+            expectations = pathStream.filter(p -> {
+                String fileName = p.getFileName().toString().toLowerCase();
+                return fileName.endsWith("html") || fileName.endsWith("dot");
+            }).collect(Collectors.toList());
+        }
+        assertThat(expectations.size()).isGreaterThan(0);
+        SoftAssertions softAssertions = new SoftAssertions();
+        for (Path expect : expectations) {
+            List<String> expectLines = Files.readAllLines(expect, StandardCharsets.UTF_8);
+            Path actual = Paths.get(target, expectedPath.relativize(expect).toString());
+            List<String> actualLines = Files.readAllLines(actual, StandardCharsets.UTF_8);
+            softAssertions.assertThat(actualLines).as("%s doesn't have the expected number of lines: %s", actual.toString(), expectLines.size()).hasSameSizeAs(expectLines);
```

Comment:
@npetzall I'm afraid that each time when we change HTML layout we must generate the expected HTML files for this test. Otherwise expectLines will be not equal to actualLines and this test will never pass... ? Correct me if I'm wrong.

Reply:
Yepp, you are correct, we can have it disabled. I only created it to be able to do refactoring of mustachwriter. Validate that my changes didn't have any side-effects.

Reply:
@npetzall I will conditionally merge your PR as it is to the master, but if we will have the problem with `producesSameContet` test in the future I will skip this test.

Reply:
Might be good idea to disable after #237 I can look into some kind of scripto update the expected.